### DISCUSSION TITLE
Add documentation for functions and classes in PlotConv2D.kt

### DISCUSSION THREAD

**Discussion by juliabeliaeva** [Kotlin/kotlindl#381] [General]
See [PlotConv2D.kt](https://github.com/Kotlin/kotlindl/blob/45268723e6af80054de603a8d938e35f4725d8ff/visualization/src/main/kotlin/org/jetbrains/kotlinx/dl/visualization/swing/PlotConv2D.kt)

**Comment by zaleslaw** [comment]
There is no public API here. Agree it requires refactoring for the given examples.

**Comment by juliabeliaeva** [comment]
> There is no public API here.

Everything in the file is visible from the outside. If this was not intended, then the visibility should be `internal`?

**Comment by zaleslaw** [comment]
It was a bunch of helper classes for a few examples. Probably it could became a public API, but it's not at this moment.

I'm not sure about making internal, if examples will work, why not, but for me is enough that it has no public modifiers

**Comment by juliabeliaeva** [comment]
> It was a bunch of helper classes for a few examples.

Let's move them to the `examples` module then. Otherwise somebody will try to use them.

Currently the whole file is highlighted yellow because of the missing documentation. We should either move the file, change visibility or add documentation.

**Comment by zaleslaw** [comment]
I found that I used visualisation during the demos, so it could be a helper class

It's strange that the whole file is highlighted yellow because where is no 
```
kotlin {
    explicitApiWarning()
}
```
in the gradle in this module.

We requires doc only for public methods in libraries, right? And in this case we doesn't require it yet.
Let's do refactoring and solve the future of this module during the planning session not in this ticket.

**Comment by zaleslaw** [comment]
@juliabeliaeva I converted it to the discussion, let's discuss the proposal about the future of visualization module here.
